awscli/customizations/ec2addcount.py

```diff
@@ -64,9 +65,15 @@ def documentation(self):
     def add_to_parser(self, parser):
         parser.add_argument(self.cli_name, metavar=self.py_name,
                             help='Number of instances to launch',
-                            default='1')
+                            ## We will delegate the default value logic to
```

Comment:
Well, it is just my preference to always put the comment as close as possible to the thing being documented, sometimes to the parameter level.

But I have no problem at all to adjust it here.

In next PR you will see this.

    def add_to_parser(self, parser):
        # We do NOT set default value here. It will be set later by event hook.
        parser.add_argument(self.cli_name, metavar=self.py_name,
                            help='Number of instances to launch')